app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -29,84 +29,92 @@ class MainActivity : ComponentActivity() {
         setContent {
             FindMyRecyclingTheme {
                 // A surface container using the 'background' color from the theme
-                Surface(color = MaterialTheme.colors.background,
-                modifier = Modifier.fillMaxWidth()) {
+                Surface(
+                    color = MaterialTheme.colors.background,
+                    modifier = Modifier.fillMaxWidth()
+                ) {
                     RecycleSearch("Android")
                 }
             }
         }
     }
-}
-
-@Composable
-fun RecycleSearch(product: String) {
-    var recyclable by remember{ mutableStateOf("") }
-    var location by remember{ mutableStateOf("") }
-
-    Column {
-        OutlinedTextField(
-            value = recyclable,
-            onValueChange = {recyclable = it},
-            label = { Text(stringResource(R.string.recyclable)) },
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
-                .border(width = 1.dp,
+
+
+    @Composable
+    fun RecycleSearch(product: String) {
+        var recyclable by remember { mutableStateOf("") }
+        var location by remember { mutableStateOf("") }
+
+        Column {
+            OutlinedTextField(
+                value = recyclable,
+                onValueChange = { recyclable = it },
+                label = { Text(stringResource(R.string.recyclable)) },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
+                    .border(
+                        width = 1.dp,
```

Comment:
What's the nature/rationale for this change?

Reply:
There was no big change but there had been bracket misplacement in the code which kept the MainActivity from being built. So there may be a big change as seen but in reality it is minor just getting the curly braces in the right places for the project to compile.